setup.py

```diff
@@ -270,6 +267,35 @@ def check_env(build):
     version += commit_hash
 
 llvm_url = f"https://github.com/llvm/llvm-project/commit/{commit_hash}"
+
+build_temp = Path.cwd() / "build" / "temp"
+if not build_temp.exists():
+    build_temp.mkdir(parents=True)
+
+data_files = [
+    (
+        "bin",
+        list(
+            map(
+                str,
+                [
+                    # this is to satisfy windows which flips the slash
+                    build_temp
+                    / "bin"
+                    / (
+                        "mlir-translate"
+                        + (".exe" if platform.system() == "Windows" else "")
+                    ),
+                    build_temp
+                    / "bin"
+                    / ("mlir-opt" + (".exe" if platform.system() == "Windows" else "")),
+                ],
+            )
+        ),
```

Comment:
Just thought I'd comment in case you find list comprehensions neat, feel free to ignore :)
```suggestion
        [
            str(build_temp / "bin" / executable / suffix)
            for executable in ("mlir-translate", "bin")
        ]
```
where suffix is calculated once outside of the loop